packages/services/api/src/shared/mappers.ts

```diff
@@ -1,384 +0,0 @@
-  supergraph: null | {
-    ownedByServiceNames: Array<string> | null;
-  };
-}>;
-
-export type UserConnection = readonly User[];
-export type MemberConnection = readonly Member[];
-export type ActivityConnection = readonly ActivityObject[];
-export type OrganizationConnection = readonly Organization[];
-export type ProjectConnection = readonly Project[];
-export type TargetConnection = readonly Target[];
-export type SchemaConnection = readonly Schema[];
-export type TokenConnection = readonly Token[];
-export type OperationStatsValuesConnection = ReadonlyArray<
-  Omit<OperationStatsValues, 'duration'> & { duration: DurationValues }
->;
-export type ClientStatsValuesConnection = readonly ClientStatsValues[];
-export type SchemaComparePayload = SchemaCompareResult | SchemaCompareError;
-
-export type SchemaCompareError = {
-  error: SchemaBuildError;
-  result?: never;
-};
-
-export type SchemaCompareResult = {
-  error?: never;
-  result: {
-    schemas: {
-      before: string | null;
-      current: string;
-    };
-    changes: Array<SchemaChangeType>;
-    versionIds: {
-      before: string | null;
-      current: string;
-    } | null;
-  };
-};
-
-export type SingleSchema = SingleSchemaEntity;
-export type PushedCompositeSchema = PushedCompositeSchemaEntity;
-export type DeletedCompositeSchema = DeletedCompositeSchemaEntity;
-export type Schema = SingleSchema | PushedCompositeSchema;
-
-export interface OperationsStats {
-  organization: string;
-  project: string;
-  target: string;
-  period: DateRange;
-  operations: readonly string[];
-  clients: readonly string[];
-}
-
-export interface SchemaCoordinateStats {
-  organization: string;
-  project: string;
-  target: string;
-  period: DateRange;
-  schemaCoordinate: string;
-}
-
-export interface ClientStats {
-  organization: string;
-  project: string;
-  target: string;
-  period: DateRange;
-  clientName: string;
-}
-
-export interface DurationValues {
-  p75: number | null;
-  p90: number | null;
-  p95: number | null;
-  p99: number | null;
-}
-
-export type TargetsEstimationDateFilter = {
-  startTime: Date;
-  endTime: Date;
-};
-
-export type TargetsEstimationFilter = TargetsEstimationDateFilter &
-  (
-    | {
-        target: string;
-      }
-    | {
-        organization: string;
-      }
-  );
-
-export type AdminStats = {
-  period: {
-    from: string;
-    to: string;
-  };
-  resolution: number;
-};
-
-export type SchemaCoordinateUsageTypeMapper =
-  | {
-      isUsed: true;
-      total: number;
-      usedByClients: () => PromiseOrValue<Array<string>>;
-      period: DateRange;
-      organization: string;
-      project: string;
-      target: string;
-      coordinate: string;
-    }
-  | {
-      isUsed: false;
-      total: number;
-      usedByClients: () => Array<string>;
-    };
-
-export type DocumentCollectionConnection = ReadonlyArray<DocumentCollection>;
-export type DocumentCollectionOperationsConnection = ReadonlyArray<DocumentCollectionOperation>;
```

Comment:
`DocumentCollectionConnection` and `DocumentCollectionOperationsConnection ` don't seem to be used as mappers. 🤔 

Originally introduced [here](https://github.com/kamilkisiela/graphql-hive/pull/1610/files#diff-071a966f8bab45a25bdc33291ac1709ab6cbe53410a66fd9060efae6d66ec4f4R237-R238) but  (1) they are not being referenced anywhere and (2) the mappers are declared as [PaginatedDocumentConnections](https://github.com/kamilkisiela/graphql-hive/pull/1610/files#diff-14af0f654ea4e4629ccf03a14bbdcccdf601b378d1cc7c4645ca3386fa0a7fbaR118). So I think this could be removed.